src/Query/EloquentQueryBuilder.php

```diff
@@ -122,4 +398,21 @@ protected function selectableColumns($columns = ['*'])
 
         return empty($selected) ? ['*'] : $selected;
     }
+
+    public function prepareValueAndOperator($value, $operator, $useDefault = false)
+    {
+        if ($useDefault) {
+            return [$operator, '='];
+        } elseif ($this->invalidOperatorAndValue($operator, $value)) {
+            throw new InvalidArgumentException('Illegal operator and value combination.');
```

Comment:
Missing `use` for this, so it's expecting `Statamic\Query\InvalidArgumentException`